app/views/planning_applications/_assessment_dashboard.html.erb

```diff
@@ -14,17 +14,7 @@
     <h2 class="govuk-heading-m" style="margin-bottom: 75px;">
```

Comment:
Just looking at this locally, I think we can remove this manual margin-bottom now?